test/second_q/formats/fcidump/test_methods_fcidump.py

```diff
@@ -67,50 +66,20 @@ def test_oh(self):
         result = self._run_fcidump(fcidump)
         self._assert_energy(result, "oh")
 
-    def test_lih_freeze_core(self):
-        """LiH freeze core test"""
-        with self.assertRaises(QiskitNatureError) as ctx:
-            fcidump = FCIDump.from_file(
-                self.get_resource_path("test_fcidump_lih.fcidump", "second_q/formats/fcidump")
-            )
-            result = self._run_fcidump(fcidump, transformers=[FreezeCoreTransformer()])
-            self._assert_energy(result, "lih")
-        msg = (
-            "'The provided ElectronicStructureProblem does not contain an `ElectronicBasisTransform`"
-            " property, which is required by this transformer!'"
-        )
-        self.assertEqual(msg, str(ctx.exception))
-
-    def test_oh_freeze_core(self):
-        """OH freeze core test"""
-        with self.assertRaises(QiskitNatureError) as ctx:
-            fcidump = FCIDump.from_file(
-                self.get_resource_path("test_fcidump_oh.fcidump", "second_q/formats/fcidump")
-            )
-            result = self._run_fcidump(fcidump, transformers=[FreezeCoreTransformer()])
-            self._assert_energy(result, "oh")
-        msg = (
-            "'The provided ElectronicStructureProblem does not contain an `ElectronicBasisTransform`"
-            " property, which is required by this transformer!'"
-        )
-        self.assertEqual(msg, str(ctx.exception))
-
-    @unittest.skip("Skip until FreezeCoreTransformer supports pure MO cases.")
-    def test_lih_with_atoms(self):
-        """LiH with num_atoms test"""
+    def test_lih_with_active_space(self):
+        """LiH with active space test"""
         fcidump = FCIDump.from_file(
             self.get_resource_path("test_fcidump_lih.fcidump", "second_q/formats/fcidump"),
         )
-        result = self._run_fcidump(fcidump, transformers=[FreezeCoreTransformer()])
+        result = self._run_fcidump(fcidump, transformers=[ActiveSpaceTransformer(4, 6)])
```

Comment:
Back when the `FCIDump` was a `driver.second_quantization`, it was valid to provide a list of atoms to it, enabling the use of the `FreezeCoreTransformer`. Instead, I am now specifying a full-space `ActiveSpaceTransformer`, which also results in the same energy as the total system